packages/lib/gpcircuits/circuits/list-membership.circom

```diff
@@ -0,0 +1,36 @@
+pragma circom 2.1.8;
+
+include "circomlib/circuits/comparators.circom";
+
+/**
+ * Module for checking whether a value is a member of a given list.
+ * A value may be either a POD entry value hash or a combination
+ * of such values (e.g. a tuple encoded as a hash).
+ */
+template ListMembershipModule(
+    // Maximum number of valid values
+    MAX_LIST_ENTRIES
+) {
+    // Value hash to be checked.
+    signal input valueHash; 
```

Comment:
The name is accurate for our intended use but at this level is there any requirement for this to be a hash?  It's just a value to compare to the list's values.